framework/Web/UI/WebControls/TStyle.php

```diff
@@ -174,6 +174,32 @@ public function setBorderWidth($value)
 		}
 	}
 
+	/**
+	 * @return string the border radius of the panel
+	 * @since 4.2.0
+	 */
+	public function getBorderRadius()
+	{
+		return $this->_fields['border-radius'] ?? '';
+	}
+
+	/**
+	 * @param string $value the border radius of the panel
+	 * @since 4.2.0
+	 */
+	public function setBorderRadius($value)
+	{
+		if (trim($value) === '') {
+			unset($this->_fields['-moz-border-radius']);
+			unset($this->_fields['-webkit-border-radius']);
+			unset($this->_fields['border-radius']);
+		} else {
+			$this->_fields['-moz-border-radius'] = $value;
```

Comment:
Do we really need -moz-border-radius and -webkit-border-radius?
The generic border-radius is well supported even by ancient browsers: https://caniuse.com/?search=border-radius
-moz-border-radius only add compatiblity for firefox 2.x
-webkit-border radius for ancient chrome versions

Reply:
I guess I'm that old school.  You got it.

Reply:
changed